libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/AlonzoCollectInputs.hs

```diff
@@ -105,6 +106,7 @@ collectTwoPhaseScriptInputsOutputOrdering = do
           testSystemStart
           (initUTxO apf)
           (validatingTx apf)
+    scriptHash = ScriptHash "58503a1d89a21fc9fc53d6a7cccef47341175a8f47636f57ccbdca2d"
```

Comment:
Not the most elegant solution and a PITA to maintain if the script changes but I wanted to avoid the `getScriptsNeeded` and `getScriptsHashesNeeded` dance. By the way, is there an easier way to get the hash of the script here? Also, would it make sense to introduce some helpers for hashing the script (if there aren't any already)?